### DISCUSSION TITLE
GPU accelerated (SE3 motion planning) Piano Mover's problem?

### DISCUSSION THREAD

**Discussion by msrivas** [NVlabs/curobo#357] [Research Q&A]
Can one use curobo to solve the Piano mover's problem? I am wondering if there is a way to use a 6DOF free floating rigid body in place of a robot arm for motion generation?

**Comment by balakumar-s** [comment]
You can setup a urdf with 6 joints (3 for linear and 3 for angular) with very large joint limits for SE3 motion planning. Here is an example urdf: https://github.com/NVlabs/curobo/blob/main/src/curobo/content/assets/robot/franka_description/moving_gripper.urdf.

This could allow for solving piano mover's problem. Let us know if it doesn't work.

**Reply by FlyCole** [reply]
If I directly load the robot from the moving_gripper.urdf file, it seems there's no collision spheres binding with this robot, so I can't do motion generation with collision avoidance. May I ask if there's a way to do that or do I still need some other files to load?
